### ISSUE TITLE
XQuery xspec & Schematron xspec are not correctly processed

### ISSUE THREAD

**Issue by cmarchand** [xspec/xspec-maven-plugin-1#12]
It would be great if we could run XSpec that unit tests XQuery or Schematron from the plugin.
Christophe

**Comment by nkutsche** [comment]
Hi,

lately I had have three Maven projects where I used Schematron and I wanted to test them also by the xspec-maven-plugin. Unfortunatelly it still doesn't work. Searching for an alternative, I found one by writing an ant script, which just calls the ant XSpec implementation provided by the XSpec framework. It worked fine, except of the circumstance that I had to add the XSpec framework zipped to the project repository (there is no Maven dependency which includes the [build.xml](https://github.com/xspec/xspec/blob/master/build.xml)).

When I found this solution I thought, that it just needs a Java wrapper calling this ant script to make a Maven plugin of it. A few days later I have a first version which works for my projects. I just published it on github: [https://github.com/nkutsche/xspec-maven-plugin/](https://github.com/nkutsche/xspec-maven-plugin/).

This little plugin should not be a replacement for the xspec-maven-plugin. It is just an alternative for cases where the origin doesn't work.

Hope this helps someone who is waiting for this issue, like me.
